package/network/utils/iptables/Makefile

```diff
@@ -113,6 +99,7 @@ define Package/iptables-nft
 $(call Package/iptables/Default)
   TITLE:=IP firewall administration tool nft
   DEPENDS:=@IPTABLES_NFTABLES +libxtables-nft +libip4tc +IPV6:libip6tc +kmod-ipt-core +kmod-nft-compat
```

Comment:
I can't test for the next few days, but the changes look ok to me, at the first glance.

One thing that you might investigate, is the IPTABLES_NFTABLES dependency for iptables-nft. Is that symbol actually needed for libxtables?

Reply:
I moved `libnftnl` dependency to `iptables-nft`, and added `@IPTABLES_NFTABLES` to `libiptext-nft (libxtables-nft)`